app/test_scoring_system.py

```
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+"""
+Script de prueba para el sistema de scoring de postulaciones.
+Este script prueba tanto el motor de scoring como los endpoints de la API.
+"""
+
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)